recipes/rhessys/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "rhessysec" %}
+{% set version = "7.2.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DavidChoi76/rhessysec/archive/v{{ version }}.tar.gz
+  sha256: 6edf6582ca076b87c26551fa7a773d9e3de48c25
```

Comment:
```suggestion
  sha256: f7a7a26a977638fc66bd0a195fb90ca930cb956898d7baaeef249e8d311e39a0
```

Reply:
@dhirschfeld Thank you for your suggestion, but where is this number (f7a7a26a977638fc66bd0a195fb90ca930cb956898d7baaeef249e8d311e39a0) from?
I set the URL, but even though the URL is exact, that occurred an error. So I put the sha256 number in meta.yaml.

Reply:
You download that binary and then calculate the sha256 checksum using whatever tools you have to hand:

![image](https://user-images.githubusercontent.com/881019/103043941-6aa76e00-45ca-11eb-9fab-8f73af044c85.png)